enhancements/security/security-profiles-operator.md

```diff
@@ -0,0 +1,629 @@
+---
+title: security-profiles-operator
+authors:
+  - "@jaormx"
+  - "@jhrozek"
+  - "@saschagrunert"
+reviewers:
+  - "@ashcrow"
+  - "@mrunalp"
+approvers:
+  - TBD
+  - "@oscardoe"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced|informational
+see-also:
+  - "/enhancements/this-other-neat-thing.md"
+replaces:
+  - [selinux-operator proposal](https://github.com/openshift/enhancements/pull/327)
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Security Profiles Operator Integration in OpenShift
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Let's integrate the [Security Profiles Operator](https://github.com/kubernetes-sigs/security-profiles-operator)
+as part of the OpenShift platform in order to allow folks to:
+
+* Easily install security profiles to secure their workloads
+  (both SELinux and Seccomp)
+
+* Manage the lifecycle of the aforementioned profiles (make sure
+  that they get attached to the appropriate container as upgrades
+  happen)
+
+* Enable users to automatically generate profiles for their workloads
+
+* Provide predefined profiles for existing default workloads in OpenShift
+
```

Comment:
I miss a very clear statement about dev-preview/tech-preview/GA. The API is alpha upstream. What is the timeline for a non-alpha API ? As alpha I think this can be a dev-preview at best.

Reply:
@sttts that's why this is marked as WIP in the title. Stuff like this needs to be determined. having it as dev-preview in 4.10 sounds acceptable.